--- includes/functions.php	2021-04-27 12:20:02.000000000 -0700
+++ includes/functions.php.new	2021-07-29 11:47:38.796736623 -0700
@@ -2996,7 +2996,7 @@
 
 			// Check the error reporting level and return if the error level does not match
 			// If DEBUG is defined the default level is E_ALL
-			if (($errno & ($phpbb_container->getParameter('debug.show_errors') ? E_ALL : error_reporting())) == 0)
+			if (($errno & ((is_null($phpbb_container) || $phpbb_container->getParameter('debug.show_errors')) ? E_ALL : error_reporting())) == 0)
 			{
 				return;
 			}
